content/en/observability_pipelines/troubleshooting.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Troubleshooting
+disable_toc: false
+---
+
+## Overview
+
+If you experience unexpected behavior with Datadog Observability Pipelines (OP), there are a few common issues you can investigate, and this guide may help resolve issues quickly. If you continue to have trouble, reach out to [Datadog support][1] for further assistance.
+
+## View Observability Pipelines Worker stats and logs
+
+To view information about the Observability Pipelines Workers running for an active pipeline:
+
+1. Navigate to [Observability Pipelines][2].
+1. Select your pipeline.
+1. Click the **Workers** tab to see the Workers' memory and CPU utilization, traffic stats, and any errors.
+1. To view the Workers' statuses and versions, click the **Latest Deployment & Setup** tab.
+1. To see the Workers' logs, click the cog at the top right side of the page, then select **View OPW Logs**. See [Logs Search Syntax][3] for details on how to filter your logs. To see logs for a specific Worker, add `@op_work.id:<worker_id>` to the search query.
+
+## Inspect events sent through your pipeline to identify setup issues
```

Comment:
I wonder if this section should start out by saying something like "If you can access your OPW instances locally..."  It jumps from stuff in the UI in `View Observability Pipelines Worker stats and logs` to this section abruptly I think.  We're now assuming the user is local to where OPW is installed.